src/api/settings.js

```diff
@@ -0,0 +1,22 @@
+import { getInstance } from 'd2'
+import { onError } from './index'
+import { SYSTEM_SETTINGS } from '../modules/settings'
+
+export const apiFetchSystemSettings = () => {
+    const endPoint = '/systemSettings'
+    const url = `${endPoint}?${SYSTEM_SETTINGS.map(s => `key=${s}`).join('&')}`
+
+    return getInstance()
+        .then(d2 => d2.Api.getApi().get(url))
+        .catch(onError)
+}
+
+export const apiFetchUserSettings = () => {
```

Comment:
This function doesn't seem to be used?

Reply:
Agreed. Just left over from importing from data-visualizer-app. Removing.